common/views/components/CaptionedImage/CaptionedImage.config.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import Image from '../Image/Image';
+import ReactDOMServer from 'react-dom/server';
+
+export const name = 'Captioned image';
+export const handle = 'captioned-image';
+export const status = 'graduated';
+
+const imageString = ReactDOMServer.renderToString(<Image
+  contentUrl='https://wellcomecollection.files.wordpress.com/2016/10/featured-image-dans-blog2.jpg'
+  caption='An edited frame from Animating the Body'
+  width={1380}
+  height={1104}
+  alt='animating the body'
+  lazyload={true}
+/>);
```

Comment:
Could you leave a note saying why we do this, and potentially a note to clean it up?